examples/js/renderers/WebGLDeferredRenderer.js

```diff
@@ -71,20 +88,34 @@ THREE.WebGLDeferredRenderer = function ( parameters ) {
 
 		var antialias = parameters.antialias !== undefined ? parameters.antialias : false;
 
-		initPassFXAA();
+		initDepthTexture();
+
 		initPassNormalDepth();
 		initPassColor();
 		initPassLight();
+		initPassReconstruction();
 		initPassFinal();
 
 		_this.setSize( _width, _height );
 		_this.setAntialias( antialias );
+		_this.enableLightPrePass( false );
 
 	};
 
-	var initPassFXAA = function () {
-
-		_passFXAA = new THREE.ShaderPass( THREE.FXAAShader );
+	var initDepthTexture = function () {
```

Comment:
No need to do it for this PR, but it's better to use `function initDepthTexture() { ... }`. That way if the code breaks we can see the name of the function in the stacktrace.